packages/opentelemetry-cloud-monitoring-exporter/test/transform.test.ts

```diff
@@ -144,7 +145,36 @@ describe('transform', () => {
   });
 
   describe('TimeSeries', () => {
-    it('should return a Google Cloud Monitoring Metric', () => {
+    const mockedAwsResource = {
```

Comment:
Optional, nit: I would go with mockAwsResource or even just awsResource, unless convention is `mocked` - mock just reads better to me and is equivalently clear in its meaning, omitting mocked altogether seems fine to me as well since we're in a test file and the implication is that these variables are for testing and no meaning would be lost in its brevity.